### DISCUSSION TITLE
David Caesar | dput

### DISCUSSION THREAD

**Discussion by dvcsr** [onlyphantom/llm-python#32] [Practicum: Generative AI]
My favorite part is when AI started to give further elaboration often at the end of the answer (_ex:_ note that...... ; This might be because there was no trading activity yesterday..., etc) for further information and clarification. That's when i am satisfied and feel enough to make edit to system prompt. 

Thank you so much for the opportunity to learn kak sam! any feedback would be very welcomed!

Exercise 1&2

result shown executed at August, 4th 2024
https://colab.research.google.com/drive/181OhvVDtrOUIE0Tgg1J6ehGffpFluUx9#scrollTo=i_SdqPeOZkH1&uniqifier=4

**Comment by onlyphantom** [comment]
Excellent work David; very neat, and even more impressive that you've also done both Challenge 1 and 2 even though only one is required. 

Thank you for be an excellent participant! Marked as passed and will be issuing a certificate to you!
